bitwarden_license/src/Commercial.Core/AdminConsole/Services/ProviderService.cs

```diff
@@ -611,8 +619,13 @@ private async Task<bool> HasConfirmedProviderAdminExceptAsync(Guid providerId, I
         return confirmedOwnersIds.Except(providerUserIds).Any();
     }
 
-    private void ThrowOnInvalidPlanType(PlanType requestedType)
+    private void ThrowOnInvalidPlanType(PlanType requestedType, bool consolidatedBillingEnabled)
```

Comment:
Non-blocking suggestion: could just default this parameter to `false` to simplify extra code changes (ex: in the `AddOrganization` method above)

Reply:
Added! https://github.com/bitwarden/server/pull/3977/commits/051e1418cc2078c8f61b037b493ed7e273c72178